app/code/core/Mage/Adminhtml/Block/Promo/Widget/Chooser/Daterange.php

```diff
@@ -66,7 +66,8 @@ protected function _toHtml()
         $form = new Varien_Data_Form();
         foreach ([
             'from' => Mage::helper('adminhtml')->__('From'),
-            'to'   => Mage::helper('adminhtml')->__('To')] as $key => $label) {
+            'to'   => Mage::helper('adminhtml')->__('To')] as $key => $label
+        ) {
```

Comment:
A new line is missing after `('To')`?

Reply:
No. (?)

Reply:
I would say:
```php
        foreach ([
            'from' => Mage::helper('adminhtml')->__('From'),
            'to'   => Mage::helper('adminhtml')->__('To')
        ] as $key => $label) {
```
Or with a variable before, but this it not important.

Reply:
This is invalid for a sniff (cant remember right now)